modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -833,6 +833,23 @@ export default {
         ...this.context,
         modified: false
       };
+      if (this.contextStack.length) {
+        // Pushed contexts might edit any property of the doc
+        this.original = klona(result.doc);
+      } else {
+        // On-page we only edit areas
```

Comment:
This logic exists for parity with our behavior when first loading the context document and populating this.original based on the areas actually in the page.